docs/source/daml/reference/interfaces.rst

```diff
@@ -70,8 +65,7 @@ Interface View Type
    :start-after: -- INTERFACE_VIEWTYPE_BEGIN
    :end-before: -- INTERFACE_VIEWTYPE_END
 
-- All implementing templates must define a special ``view`` method which returns
-  a value of type declared by ``viewtype``.
+- All interface instances must implement a special ``view`` method which returns a value of the type declared by ``viewtype``.
```

Comment:
Is there some change here that I'm missing?

Reply:
also replaced `implementing templates` with `interface instances`